images/js/main.js

```diff
@@ -28,9 +28,11 @@ const start = async () => {
   window._finished = true; // eslint-disable-line no-underscore-dangle
 };
 
-try {
-  start();
-} catch (err) {
-  console.log('ERROR'); // eslint-disable-line no-console
-  console.log(err.message); // eslint-disable-line no-console
-}
+window.startProcess = () => {
+  try {
+    start();
```

Comment:
Since start is async, then startProcess should also be async and this line should be 'await start()'.

Reply:
fixed!